PR: Add some info regarding command line arguments for the Windows installer (#17728)
spyder-ide/spyder: installers/Windows/README.md

Mu is distributed under the GPL-3.0 license.

We use here the base functions of the script to build a Windows installer of
spyder, mainly changing the code to generalize its functionality and configure
the run with more arguments (package name, conda python interpreter, extra
packages, etc.).

> As a NSIS installer the generated installer counts with some command line flags:

Reviewer: ```suggestion
As an NSIS installer, the generated Spyder executable has the following command line flags:
```

https://nsis.sourceforge.io/Docs/Chapter3.html#installerusage

> Besides those flags, the installer also has with the following command line options

Reviewer: ```suggestion
Besides those flags, the installer also has the following command line options
```

This was my mistake, sorry.

> Besides those flags, the installer also counts with the command line options
> available when using the `MultiUser.nsh` header file:

Reviewer: ```suggestion
Besides those flags, the installer also has with the following command line options
available because it uses the `MultiUser.nsh` header file:
```

I'm not so sure about this suggestion, but that's what I understand from what you wrote.
